### DISCUSSION TITLE
F-norm

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [idaholab/moose#23851] [Q&A General] [answered]
Hello all，
I have two questions:

1.Can the user provide the preprocessor pmat?
2.How to output matrix norm?
thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please don't post twice the same question. I'm deleting the other one.
These answers are likely in the petsc manual, you can pass arguments to petsc

Guillaume

**Reply by nanguaxiaofendui** [reply]
thank you
